.github/workflows/build-steps.yml

```diff
@@ -13,6 +13,7 @@ on:
       - '**.md'
       - '**/.vscode/*.json'
       - 'samples/**'
+      - 'change/**'
```

Comment:
Are we sure we want to ignore change-files? If there is a new change-file shouldn't we assume that a build should be run?

Reply:
The presence of a changefile on its own should not trigger a build. The presence of changed files in the libs/samples/workflow should be what triggers a new build

Reply:
Sorry, not samples in this case, but in E2E case

Reply:
Makes sense